blint/data/annotations/review_rootkits_win.yml

```diff
@@ -0,0 +1,178 @@
+---
+text: Review for Windows rootkits
+group: METHOD_REVIEWS
+exe_type:
+  - x86_64-executable
+rules:
+  - id: BANSHEE
+    title: Detect Banshee
+    summary: rootkit
```

Comment:
I think the summary is shown in the table print via the cli. Best to make it a few capitalized words.